site/src/App.scss

```diff
@@ -37,7 +37,7 @@ code {
 
 .app-body {
   display: flex;
-  height: calc(100% - 72px);
```

Comment:
This is a result of a different layout shift which is fixed in #865 - we should revert this